src/index.js

```diff
@@ -13,16 +13,19 @@ const HorizontalCalendar = ({
 	date = new Date(),
 	choosed = 1,
 	isDarkWeekend = true,
+	mondayFirst = false, 
 	platform,
 	onClick,
 }) => {
 	const baseClassNames = getClassName("Card", platform);
 
+	const adays = mondayFirst ? daysMondayFirst() : days() 
+
 	const onItemClick = (e) => {
```

Comment:
Get up functions days and daysMondayFirst to avoid TypeError. declaring adays should be lower then them.

Reply:
Done.